scss/titles.scss

```diff
@@ -0,0 +1,18 @@
+$title-decoration-height: 90px;
+$title-decoration-width: 70px;
+
+.title {
+  font-size: $text-big;
+  margin: 128px 160px 48px;
```

Comment:
This margin is quite wierd, you should do something like `margin: 150px` and no more. At least for now